src/plot/plaxis.cpp

```diff
@@ -694,17 +705,18 @@ void wxPLTimeAxis::RecalculateTicksAndLabel() {
         // Less than 2 days.  Need to label time.
 
         wxDateTime timeKeeper2(timeKeeper.GetTicks());
-        timeKeeper2.Add(wxTimeSpan::Minutes(60 * world_len));
+        timeKeeper2.Add(wxTimeSpan::Minutes(60 * world_len)); // TODO 289 - check scaling
```

Comment:
Is this TODO done? If so can you remove the comment?

Reply:
updated in https://github.com/NREL/wex/pull/179/commits/9f31b487cd0f51e1de6726f362e596e503eb157c